libs/nx-remotecache-s3/src/lib/util.ts

```diff
@@ -10,3 +13,14 @@ export const buildCommonCommandInput = (
   Key: `${prefix}${filename}`,
   /* eslint-enable @typescript-eslint/naming-convention */
 });
+
+export const isReadOnly = (
```

Comment:
Maybe the type of this commit is more a fix than a feat? I would edit the commit message to be something like `fix(nx-remotecache-s3): readonly not correctly calculated`

Reply:
Ok I will amend it